code/src/java/pcgen/core/BioSet.java

```diff
@@ -250,14 +251,7 @@ public void removeFromUserMap(final String region, final String race, final Stri
 		final String key;
 		final int x = tag.indexOf(':');
 
-		if (x < 0)
-		{
-			key = tag;
-		}
-		else
-		{
-			key = tag.substring(0, x);
-		}
+		key = x < 0 ? tag : tag.substring(0, x);
```

Comment:
This type of in-lining is controversial of whether it is a best practice.  Before we make these types of changes, we need a discussion on the code list as to best practices.  I happen to dislike this usage because the "x < 0" is not in parenthesis and thus the line is harder to interpret.